lib/remi/data_subjects/file_system.rb

```diff
@@ -50,6 +50,9 @@ def initialize(*args, **kargs, &block)
     attr_reader :most_recent_only
     attr_reader :group_by
     attr_reader :most_recent_by
+    attr_reader :n_hours_one
```

Comment:
These parameters should be documented above (we use YARD).  

It's also not obvious what the difference is between "one" and "max".  Can we make the parameters a little clearer?

Actually, how about using only one new parameter, call it `created_within`.  Let the argument to that parameter be the number of seconds, so that we can make use of ActiveSupport's syntax:  `created_within: 3.hours` or `created_within: 7.days`.  Then, reuse the parameter `most_recent_only` (T/F) to determine whether we collect all files within that range, or just the most recent one.